README.md

```diff
@@ -185,33 +191,35 @@ threescale_portal_endpoint: <THREESCALE_PORTAL_ENDPOINT>
 # Comma separated list of services (Id's or system names)
 # If empty, all available services will be used
 threescale_services: ""
+
+injector_traffic_path: "<fully-qualified-path>/deployment/benchmarks"
```

Comment:
I do not understand. 

What does the user has to put in that `fully-qualified-path`? There is no way to get that from ansible? The user will be confused

Reply:
I left default value `"deployment/benchmarks"` and I get:

```
Destination directory deployment/benchmarks does not exist
```

I tried with fully qualified local name and does not either work with the same error.

If the fully qualified name refers to injector host directory path, you could use `tmp` or something like that. Or have ansible create directory for you. There are lots of options

Reply:
I've tried a variety of relative paths and they never work. Instead I've only found the absolute path to work.

Reply:
You cannot ask users to specify fully qualified path in all the injector nodes. Ansible should be able to make it for them. Or find another way.

Reply:
I've found another way as you suggested. The user does not need to specify a path. Paths are configured now to be relative to the perftest-toolkit.
 Please check the replacement meets your expectations.